doc/dev/developer_guide/running-tests-in-cloud.rst

```diff
@@ -124,8 +124,7 @@ uploaded to http://teuthology-logs.public.ceph.com.
 Run a standalone test
 ---------------------
 
-The standalone test explained in `Reading a standalone test`_ can be run
-with the following command
+The standalone test can be run with the following command
```

Comment:
wondering if this part about ceph-workbench is relevant anymore - we can remove it in a follow-on PR

Reply:
@neha-ojha https://tracker.ceph.com/issues/48942